### PR TITLE
Blk 698 hardhat tests

### PR DESCRIPTION
**Pull request by bpbirch**

### CODE REVIEW

File: src/tasks/logout.ts [hunk 1]
```diff
@@ -7,10 +7,10 @@ import {
 import {default as chalk} from 'chalk';
 
 /**
- * deletes auth token from configstore (authconfig.json)
```
Comment (JehoG) [inline src/tasks/logout.ts:10]: We want to remove this line also?

File: src/tests/integration/test_deploy.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import {
+    SimbaConfig,
+} from "@simbachain/web3-suites";
+import deployContract from "../../tasks/deploycontract";
+import { expect } from 'chai';
+// import { execSync } from "child_process";
+import {default as chalk} from 'chalk';
+import axios from "axios";
+// import robot from "robotjs";
```
Comment (JehoG) [inline src/tests/integration/test_deploy.ts:9]: Many forgotten commented lines?

**Review by JehoG — APPROVED** [review]

File: src/tasks/logout.ts (hunk 1 above)
Comment (bpbirch) [reply src/tasks/logout.ts:10]: fixed

File: src/tests/integration/test_deploy.ts (hunk 2 above)
Comment (bpbirch) [reply src/tests/integration/test_deploy.ts]: fixed

**Pull request merged**